### ISSUE TITLE
[Tegumento] Unlock External Account Linking

### ISSUE THREAD

**Issue by rachaelwoolen** [seedco/Workshop#17]
### What
Ability for Support to unlock trial deposit amounts when customer is locked out. Do not trigger new deposits. 

### Description
When linking an external account by trial deposit this is the current flow: 

1. enter a wrong value, get message `Sorry the amounts you entered are not correct. Please check your account and try again`
2. enter a wrong value, get message `Sorry the amounts you entered are not correct. We have reissued new amounts. Please check your account and try again`
3. enter a wrong value, get message `Sorry the amounts you entered are not correct. Please check your account and try again`
4. enter a wrong value, get message `Too many failed attempts. Please contact support`
… and all attempts afterwards will show the `contact support` message

Proposed Flow: 

1. Customer enters wrong value, gets message: Sorry the amounts you entered are not correct. Please check your account and try again`
2. Enters wrong value again, gets same message.
3. Enters wrong value a 3rd time, gets message: `Too many failed attempts. Please call support` populate phone number
4. Support requires customer to call, verifies their identity and that they own the account they are attempting to link. Unlocks trial deposit field directly from Teg.
5. If customer locks themselves out a second time, require them to send a statement verifying that they own the external account they are trying to link 

### User Story
As a busy business person, I don't have time to read instructions when linking an external account. I just need to get it linked so I can transfer funds right away. If I enter the wrong trial deposits three times and contact support, I expect results. 

As a company, we want customers to link their external accounts and fund their Seed account asap so that they can become active customers. It is highly unlikely that someone could guess the trial deposit amounts in 3 tries. By controlling the unlock, we ensure that customers can link their accounts quickly, while also getting a chance to verify them, explain what trial deposits are, how they work and hopefully identify whether or not they are potential fraudsters. 

cc @lh12345 @melgeyer @btmerr

**Comment by melgeyer** [comment]
If @lh12345 approves from a compliance perspective this new flow makes a lot of sense to me and I prefer it - both from security and customer activity perspectives. 

My only question is, can this function of reaching out to support to reset their attempts at entering trial deposits, also be done via Zendesk message - where there is no IDV and can be accessed via customer's email (and whatever their own personal security habits happen to be)?

**Comment by lh12345** [comment]
@rachaelwoolen @melgeyer Hey ladies - can y'all clarify the proposed step 4 - besides ID verification, what else is being asked to ensure that "they own the account that they are attempting to link" before unlocking the TD field? Does the step include asking the customer to provide the 2 existing TD amounts again or simply unlocking the TD field after ID verification and telling them to retry via app again?

**Comment by melgeyer** [comment]
@lh12345 we're not removing the requirement for the user to enter in trial deposits. They still have to do this work. 

The current process has Seed sending new trial deposits to the other account automatically after a few miskeys. 

We are hoping that instead of doing the current flow - which extends the timeline (user now has to wait another 3 to 5 days to get the new trial deposits) & doesn't educate the user on what is actually expected of them. 

When someone miskeys the trial deposits, it is generally for one of two reasons: 
1. They are a valid customer who has access to the other account, but they don't understand how the whole process works and need education on it: This is very common b/c people don't read. So having them call support lets us ID verify the customer and then reset the trial deposit entries - which simply lets them enter in the numbers. We don't provide the trial deposit numbers - we don't even know them. We just walk them through what needs to happen & give them a chance to try again.

2. Customer who doesn't actually have access to the other account: This user is out of luck entirely with this new flow. After they get through IDV, we'd say: Ok, your link was locked because you didn't enter the correct trial deposits. We do trial deposits b/c you have to prove that you have ownership of the other account by confirming small amounts that we've sent to you. Now that we've unlocked the feature again for you, you need to go to that other account, look up the trial deposits and enter them into our app (folks don't recognize trial deposits.. mostly b/c they don't read and understand what they're looking for). If you don't have access to the other account then you can't link these two accounts. 

does this clarify?

**Comment by lh12345** [comment]
@melgeyer yep - thanks for the helpful clarification. i'm guessing bus. customers are more familiar with TDs than consumers, so hopefully these kind on inquiries won't occur too often. let's move forward with the new proposed process - thanks!

**Comment by melgeyer** [comment]
Excellent, Thanks, @lh12345! I hope you're right that business customers are more educated on this b/c explaining it is a pain and can be time-consuming.
